src/collectHeatmapClicks.js

```diff
@@ -16,6 +16,19 @@ export default (subscribe, { interval = 100 }) => {
     }
   };
 
+  const performOnScriptEnd = function(fnc) {
+    setTimeout(fnc, 0);
+  };
+
+  const filterClasses = (classes) => {
+    let filteredClasses = classes;
+    blacklistedClasses.forEach(rule => {
+      filteredClasses = filteredClasses.replaceAll(rule, ' ').trim();
```

Comment:
That wont work on IE, that's ok with you? :P

Reply:
Thanks for tip! I will check what we can do with it :)

Reply:
forEach too, not only for replaceAll